src/main/java/com/org/jenkins/custom/jenkins/distribution/service/generators/PackageConfigGenerator.java

```diff
@@ -13,7 +13,7 @@
 
     private final static UpdateCenterService UPDATE_SERVICE = new UpdateCenterService();
 
-    public static final String generatePackageConfig(final JSONObject formData)
+    public static String generatePackageConfig(final JSONObject formData)
```

Comment:
Was there a reason this is no longer final?

Reply:
Yeah I was testing a few methods, switching back to final.